src/Stan/Core/Toggle.hs

```diff
@@ -0,0 +1,25 @@
+{- |
+Copyright: (c) 2020 Kowainik
+SPDX-License-Identifier: MPL-2.0
+Maintainer: Kowainik <xrom.xkov@gmail.com>
+
+Data types to show/hide different parts of inspections and observations.
```

Comment:
This is a very nice module! I can see how it can be even more useful in the future :+1: